### ISSUE TITLE
Developer Documentation and Portal

### ISSUE THREAD

**Issue by Kisgus** [LinkdropProtocol/linkdrop-monorepo#79]
# Description

We need a cover page and developer portal set up to make linkdrop accessible and easy to use for developers within their own projects

# Subtasks included

- [ ] Draft text to developer portal - e.g what can developers use it for right now
- [x] Do mock-ups of developer portal
- [x] Linkdrop Gitbook MVP
- [ ] Integrate Ready Mage developer portal on linkdrop website

# Definition of done

The developer portal and developer documentation is ready to be announced

# Reviewer

@Kisgus

**Comment by seichris** [comment]
What do devs want to know?

Developers who want to know how to integrate e.g. Linkdrop 1-1 into their wallet.
Developers already signed in to Linkdrop PRO, looking for ways to generate airdrops in their code.
Developers who want to help with our protocol

**Comment by Kisgus** [comment]
Updated the issue. After #63 #80 are done, please start on this @seichris

**Comment by Kisgus** [comment]
https://www.figma.com/file/cAZC7IPUn1N1soraUWLmIxCw/Linkdrop-R%26D?node-id=477%3A0

**Comment by Kisgus** [comment]
Deferred
